charter/charter.md

```diff
@@ -54,3 +54,8 @@ Defining new congestion control schemes is explicitly out of scope for
 the WG. However, new QUIC extensions that support development and
 experimentation with new congestion control schemes may fall under the
 third work area.
+
+The working group will specify how QUIC stream multiplexing and other
+application-oriented extensions (e.g. Datagram) can be adapted to work
+over a secure, reliable, and bidirectional byte stream substrate. No
+effort will be made to enable insecure substrates.
```

Comment:
Thank you for doing this.

I kind of think it'd make sense to present this as the "fourth area of work" in the numbered list above, considering that the paragraph following the numbered list talks about how applications would be mapped onto the abstractions provided by QUIC, a topic that should apply equally to QMux.

That said, I'm fine with the PR as-is.